src/Common/tests/XunitTraitsDiscoverers/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -23,7 +23,7 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
         public IEnumerable<KeyValuePair<string, string>> GetTraits(IAttributeInfo traitAttribute)
         {
             IEnumerable<object> ctorArgs = traitAttribute.GetConstructorArguments();
-            Contract.Assert(ctorArgs.Count() >= 2);
+            Debug.Assert(ctorArgs.Count() >= 2);
```

Comment:
I think Contract.Assert is just gonna throw an exception while Debug.Assert is gonna fail fast when no debugger attached. I think Resharper is reading attributes in the context of VS which could potentially make VS fail fast. Not sure about that though.